responses.json

```diff
@@ -99,10 +99,17 @@
     },
 
     "handle_sex": {
-        "response": "Thanks for thinking that way with me but, I'm gonna pass this time",
+        "response": "Thanks for thinking that way with me but, I'm gonna pass this time.",
         "list_of_words": ["do","you","want","to","have","sex","make","love"],
         "is_single_response": false,
         "required_word": ["sex"]
-    } 
+    },
+
+    "your_job": {
+        "response": "My job is to entertain and chat you all.",
```

Comment:
I think it should be "My job is to make you have some fun when chatting with me"